README.md

```diff
@@ -88,7 +88,33 @@ There is a decent guide [here](https://www.linuxtechi.com/how-to-install-debian-
 
 #### Raspberry Pi / ARM64
 
-TBD
+##### Hardware Considerations
+If you choose to use a Raspberry Pi for the cluster, I would recommend at minimum a Raspberry Pi4 (4GB) and preferably an 8GB model. Additionally, I would recommend booting from and external SSD, rather than the SD card, as slower write times will impact etcd if you use it as a master.
+
+This is supported [natively](https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#raspberry-pi-4b-and-raspberry-pi-400), however if you have an early Raspberry Pi4, you may need to update the bootloader, like [this](https://www.tomshardware.com/how-to/boot-raspberry-pi-4-usb).
+
+For SSD hardware, it's common to use the [Kingston A400](https://www.amazon.com/dp/B01N5IB20Q/), with any [USB3.0 USB SATA Adapter](https://www.amazon.com/dp/B011M8YACM).
```

Comment:
Would it be better to include a pcpartpicker url for this hardware?

Reply:
We can do that for the SSD, but not so much the. USB adapter, AFAIK.

Reply:
You can add in "other parts" I think in pcpartpicker

Reply:
Yeah, I can see external drives, but not the adapter. I'll just remove it.